### PR TITLE
Testing/results

### LINKED ISSUE #36: Get testing suites up and running with TypeScript

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- I added dependencies for using mocked from `'ts-jest/utils'` library
- I wrote a test in About that the summary is rendered on the page
- I updated the test in App to display the new information since we made changes to title and buttons
- I wrote a test in Home that the four horizontal scroll boxes render on the page
- I wrote tests in Results that the results display on the page properly
- I wrote a test in Search that the button click fires the correct method
- Closes #36 Closes #38 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
- There are still some failing tests but they will need to be updated once we get data rendering anyway so I didn't waste the time on them!

# Message for reviewer:
- You will probably have to run `npm install` again when you pull this down.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,5 +37,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/jest": "^24.9.1",
```
Comment (JoshSevy) [inline package.json:42]: Thanks for looking into the dependencies and fixing the testing suites.

File: src/Home/Home.test.tsx [hunk 2]
```diff
@@ -21,5 +20,18 @@ describe('Home', () => {
 
   // it('should fire event when card is clicked', () => {
   //   // ???? it's a link to the location page, is this a testable event?
-  // })
+	// })
```
Comment (JoshSevy) [inline src/Home/Home.test.tsx:23]: These tests look great, great work.

File: src/Results/Results.test.tsx [hunk 3]
```diff
@@ -24,19 +27,71 @@ describe('Results', () => {
 				'continent': 'North America',
 				'label': 'Miami, FL, USA'
 			}
+		],
+		mockedWorldResults = [
```
Comment (JoshSevy) [inline src/Results/Results.test.tsx:31]: I think we can put all of our minds together and crush through these async tests in no time. Great work

File: package.json (hunk 1 above)
Comment (leighlars) [reply package.json:42]: What a headache. Glad ya got it working!

File: src/About/About.test.tsx [hunk 4]
```diff
@@ -1,21 +1,23 @@
-import React from "react";
-import { render, screen, getByAltText} from "@testing-library/react";
-import About from "./About";
-import { MemoryRouter } from "react-router-dom";
+import React from 'react'
+import { render, screen } from '@testing-library/react'
```
Comment (leighlars) [inline src/About/About.test.tsx:2]: I promise i'm not putting double quotes in there on purpose!!!!!

File: src/App/App.test.tsx [hunk 5]
```diff
@@ -1,36 +1,35 @@
...
+    const logo = getByRole('heading', { name: 'Breathe Am' });
+    const searchBar = getByPlaceholderText('Search city, zip, or county')
     expect(logo).toBeInTheDocument()
     expect(searchBar).toBeInTheDocument();
   })
 
   it('should show various pages when nav links are clicked', () => {
     // passed on everything but the return to home link at bottom
-    const {getByRole} =  render(<MemoryRouter><App/></MemoryRouter>)
+    const {getByRole} = render(<MemoryRouter><App/></MemoryRouter>)
```
Comment (leighlars) [inline src/App/App.test.tsx:18]: Good work!

File: src/Home/Home.test.tsx [hunk 6]
```diff
@@ -21,5 +20,18 @@ describe('Home', () => {
...
+	
+	it('should display 4 horizontal scrolls with info cards', () => {
+		const { getByRole } = render(<MemoryRouter><Home /></MemoryRouter>)
```
Comment (leighlars) [inline src/Home/Home.test.tsx:26]: omg thank you for this!!

File: src/jest.config.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (leighlars) [inline src/jest.config.js:1]: whoa

**Review by leighlars — APPROVED** [review]
Great work!!!

File: src/About/About.test.tsx (hunk 4 above)
Comment (ErinUntermeyer) [reply src/About/About.test.tsx:2]: Oh I know! It happens to me all the time with VSCode. I just grab em when I'm in that file!

**Pull request merged**